.travis.yml

```diff
@@ -79,6 +79,10 @@ script:
       presto-product-tests/bin/run_on_docker.sh \
```

Comment:
You can use the template from here: https://github.com/Teradata/presto/commit/e9bb0ebdb1d3b8c7ccc197cf946ee783bca63826